docs/DeveloperGuide.adoc

```diff
@@ -345,6 +345,35 @@ image::UndoRedoActivityDiagram.png[width="200"]
 **Cons:** Requires dealing with commands that have already been undone: We must remember to skip these commands. Violates Single Responsibility Principle and Separation of Concerns as `HistoryManager` now needs to do two different things. +
 // end::undoredo[]
 
+// tag::RemarkCommand[]
+=== Remark mechanism
+The remark command takes in the user input through command line in the format remark INDEX /r REMARK, where INDEX is the index of the person in the addressbook and REMARK is the remark of the relationship of that person to another person. +
```

Comment:
Hmm... I believe this sentence fits more into the User Guide than into the Developer Guide. Feel free to take a look at what the others have written for this section to find out how it should be like